example/src/screens/timelineCalendar.js

```diff
@@ -0,0 +1,286 @@
+import _ from 'lodash';
+import React, {Component} from 'react';
+import {
+  Platform,
+  StyleSheet,
+  View,
+  Text,
+  TouchableOpacity,
+  Button
+} from 'react-native';
+import {
+  ExpandableCalendar,
+  Timeline,
+  CalendarProvider
+} from 'react-native-calendars';
+
+// const today = new Date().toISOString().split('T')[0];
+// const fastDate = getPastDate(3);
+// const futureDates = getFutureDates(9);
+// const dates = [fastDate, today].concat(futureDates);
+
+// function getFutureDates(days) {
+//   const array = [];
+//   for (let index = 1; index <= days; index++) {
+//     const date = new Date(Date.now() + 864e5 * index); // 864e5 == 86400000 == 24*60*60*1000
+//     const dateString = date.toISOString().split('T')[0];
+//     array.push(dateString);
+//   }
+//   return array;
+// }
+
+// function getPastDate(days) {
+//   return new Date(Date.now() - 864e5 * days).toISOString().split('T')[0];
+// }
```

Comment:
Do we need all this commented code?

Reply:
Remove Extra Commented Code and Added If Check for this.props.eventTapped.